similarity_api/src/db/db.py

```diff
@@ -0,0 +1,14 @@
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import text
+
+db = SQLAlchemy()
+
+def query_many(query_str, data):
+    try:
+        query_str = text(query_str)
+        result = db.session.execute(query_str, data).all()
+        out = [{column: value for column, value in row._mapping.items()} for row in result]
+        return out
+    except Exception as e:
+        return None
```

Comment:
can we either log or display the error on stderr so we can know when something is going wrong

Reply:
abe177d43f6a6d50d76beeb6c44d25adad3020b4